src/main/java/com/exactprosystems/remotehand/windows/actions/ClickContextMenu.java

```diff
@@ -40,8 +40,10 @@ public String run(WindowsDriverWrapper driverWrapper, Map<String, String> params
 		
 		ElementSearcher es = new ElementSearcher(params, driver, cachedWebElements);
 		WebElement element = es.searchElement();
-		
-		element.click();
+
```

Comment:
What is the reason of this changes?

Reply:
In some cases there when you execute "element.click()" on menu items, an "An element command could not be completed because the element is not pointer- or keyboard interactable" error occurs. This change fixes this issue.